### DISCUSSION TITLE
⚠️Novedades - Semana 19/03⚠️

### DISCUSSION THREAD

**Discussion by perealberto** [Aparking/AparKing_Backend#151] [Announcements]
¡Bienvenidos a las novedades de esta semana! Continuamos con el arduo camino de ISPP. Esta semana hemos salido un poco magullados 😵 de la revisión semanal. Pero que no decaiga el ánimo, que la Semana Santa 🙏🏻 ya está aquí para acolchar el golpe y respirar un poco.

Vamos a ver los avances semanales!!

## Feedback

Parece ser que tenemos la mala costumbre de no hacer bien las cosas. Por tanto, dejamos por aquí una pequeña lista con las ideas generales del feedback:

- La formula planteada da a entender que estamos dando más importancia a una gente que a otra, lo cual no mola nada, así que hay que modificarlo.
- Llevamos vendiendo el **oro y el moro** desde el primer día, pero, aún no hemos mostrado ningún prototipo. La próxima presentación tenemos que mostrar un vídeo en condiciones del sistema funcionando.
- Relacionado con lo anterior, si bien es verdad que nuestro gran presentador @sergiosant02 😎 vende el producto como nadie en el mundo, nuestras gráficas de progreso muestran todo lo contrario 💀. Por tanto, tenemos que ponernos las pilas.

## Estado del proyecto

Nos encontramos en la semana 8/15 del proyecto. Ya queda menos de la mitad de la asignatura, lo cual significa que:
- Ya queda nada para terminar.
- No podemos aflojar el ritmo.

Hemos consumido un total de 1181/2100 horas. Es primordial que ajustemos las tareas que hagamos al tiempo total disponible.

![image](https://github.com/Aparking/AparKing_Backend/assets/128732971/ac468b1d-19f0-4957-bbb9-75e453a178fd)

## Puntos de historia

Llegados a este punto... qué os voy a contar sobre el avance si vosotros lo sabéis mejor que nadie (xd). Aquí podemos ver el burndown de esta semana. En resumen, tenemos un **gran bloqueo ** con tareas del sprint 1 que no acaban de terminarse y están consiguiendo retrasar el sprint 2.

![burndownchart](https://github.com/Aparking/AparKing_Backend/assets/128732971/62a91117-affc-445e-a7d4-1f7b3038c3bc)

## Trabajo para esta semana y Semana Santa

Bien, dicho todo esto, quedan unos días de duro trabajo, aunque supongo que ya os lo habréis imaginado. No podemos llegar a la próxima presentación y tener tareas del sprint 1 aún abiertas. Luego, el [sprint 2](https://github.com/Aparking/AparKing_Backend/milestone/2) acaba el 1 de abril, es decir, el lunes después de Semana Santa. La presentación es el 2 de abril. Entonces, para estas dos semanas tenemos que hacer:

Documentación:
- software review
- customer agreement
- actualizar costes
- commitment agreement y fórmula de rendimiento
- usuarios piloto
- temas legales (pildora)
- presentación

Desarrollo:
- gestión de usuarios
- gestión de alquileres de garaje
- gestión de parking público (mapa)
- gestión de planes de pago

Además, dejo por aquí otras tareas que no son de documentación, ni desarrollo, pero igualmente tenemos que realizar:
- prototipo
- subir documentación al repositorio en markdown
- subir video de prototipo en el repositorio
- leer failure conditions (hay nuevos puntos)
- leer software review guidelines (hay nuevos puntos)
- ver píldoras (importante porque los profesores lo van a mirar) -> renombrar tiempo de las píldoras teóricas y añadir issues para las pildoras teóricas
- revisar conexión con bluejay porque no nos funciona

Ea... pos' ya estaría... poquita cosa verdaaaad? 😖 Los coordinadores sabemos que queda una semana dura, pero también sabemos que si la aprovechamos se saca todo esto y más. En fin, no hay más que añadir, que paséis una buena Semana Santa y comáis muchas torrijas 😋.

(No olvidéis reaccionar a las novedades ;) )

![gif](https://64.media.tumblr.com/ae2a5c1445e639e49df3fce512ece7dc/tumblr_nzctcgE4qY1tgsqpeo1_500.gifv)

**Comment by CarmenRuizPorcel** [comment]
Se vienen cositas 😎
No olvidemos que toda la documentación se sube directamente a docusaurus 🦖 . Vamos a intentar dejar Onedrive solo para lo esencial que no se pueda subir al docusaurus.

**Comment by MarioArocaPaez** [comment]
Oi, pessoal!

Essa semana foi puxada, né? Mas olha, a Semana Santa tá chegando pra nos dar aquele respiro. 🌼

A revisão deixou a gente meio abalado, mas é assim mesmo. Agora é hora de juntar nossas forças e seguir em frente. Temos um monte pra fazer, mas sei que, juntos, a gente consegue. Vamos com tudo nessa reta final!

Todo mundo tá se esforçando muito, e isso é demais! Vamos continuar assim, que a vitória é certa. Depois, a gente relaxa e aproveita um descanso merecido. 🙌

Força aí, pessoal! Conto com vocês pra gente fechar isso aí com chave de ouro. E que venham as torrijadas depois! 😄

**Reply by laurolmer** [reply]
k dise mano
